nvidia/packages/driver_packages/back_up.rst

```diff
@@ -0,0 +1,55 @@
+Back Up and Restore
+###################
+
+The tutorial will guide you how to back up and restore system image.
+
+.. note:: 
+
+    The target machine must be connected to **Host PC** over USB and entered into **recovery mode**.
+
++--------------------+------------+
+|Model               |Target Name |
++====================+============+
+|RQX-580             |rqx_580     |
++--------------------+------------+
+|RQX-58G             |rqx_58g     |
++--------------------+------------+
+|NPN-1 (Pico Nano)   |npn1        |
++--------------------+------------+
+|NPN-2 (Pico AGX NX) |npn2        | 
++--------------------+------------+
+
+1. Cloning the Image
+^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+    cd <driver package>
+    sudo ./flash.sh -r -k APP -G backup.img <target name> mmcblk0p1
+
+This command actually creates two image files: ``backup.img`` and ``backup.img.raw``. 
+
+| ``backup.img.raw`` contains raw data read from the target's APP partition, it has exactly the same size as APP partition. 
+| ``backup.img`` is a sparsed image, it has a much smaller size and can speed up the restoring step. 
+
+2. Copy the backup image to flashing directory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+copy the .raw file which contains complete image from source device. 
```

Comment:
Which should I copy? backup.img or backup.img.raw?

Reply:
I fix the context here.